examples/snn/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# ------------------------------------------------------------------------
+#  Gunrock: Sub-Project Shared Nearest Neighbor
+# ------------------------------------------------------------------------
+project(snn)
+message("-- Project Added: ${PROJECT_NAME}")
+include(${CMAKE_SOURCE_DIR}/cmake/SetSubProject.cmake)
+
+target_link_libraries(${PROJECT_NAME} ${CUDA_cudadevrt_LIBRARY})
+
+if (FAISS_INCLUDE_DIR)
+    include_directories(${PROJECT_NAME} ${FAISS_INCLUDE_DIR})
```

Comment:
You already do this in `SetSubProject.cmake`. Should that one be removed and just keep the one in `snn/CMakeLists.txt`? `SetSubProject.cmake` is included in here.